app/helpers/pretty-bytes.js

```diff
@@ -2,4 +2,17 @@ import { helper } from '@ember/component/helper';
 
 import prettyBytes from 'pretty-bytes';
 
-export default helper(([bytes], options) => prettyBytes(bytes, options));
+/**
+ * See https://github.com/rust-lang/crates.io/discussions/7177
+ *
+ * Here we set fraction digits to 1 because `cargo publish`
+ * uses this format (see https://github.com/rust-lang/cargo/blob/0.73.1/src/cargo/ops/cargo_package.rs#L167-L170)
+ */
+export default helper(([bytes], options) =>
+  prettyBytes(bytes, {
+    ...options,
+    binary: true,
+    minimumFractionDigits: 1,
+    maximumFractionDigits: 1,
```

Comment:
```suggestion
    binary: true,
    minimumFractionDigits: 1,
    maximumFractionDigits: 1,
    ...options,
```

this should make it possible to override these defaults